app/models/trip.rb

```diff
@@ -0,0 +1,167 @@
+require_relative '../models/subscription'
+require_relative '../models/date_format'
```

Comment:
you shouldn't have to manually require these models. they should all be required for you in `config/environment.rb`